README.md

```diff
@@ -39,3 +39,18 @@ $ pqs
 {"schema":"public","table":"notes","op":"INSERT","payload":{"created_at":"2017-09-04T01:11:34.65629","id":1,"notes":"here is an example note"}}
 {"schema":"public","table":"notes","op":"DELETE","payload":{"created_at":"2017-09-04T01:11:34.65629","id":1,"notes":"here is an example note"}}
 ```
+
+
+## redact fields
+
+If there's a need to disable sensitive fields (i.e password) that get's exported the `redactions` flag can be used with `pqsd`:
```

Comment:
can we say something like 'need to prevent sensitive fields (i.e. password) from being exported'..

Reply:
Yeah that sounds better. (I appreciate the language correction as English is not my first language)